### ISSUE TITLE
Driving slowly, despite connected hoses and cables in the newest Version 2.2.0.0

### ISSUE THREAD

**Issue by Horschde** [stijnwop/manualAttach#160]
Hallo,

i have updatet your Mod to V 2.2.0.0 (just now). I can't driving with a trailer in Fullspeed, It drives as if the cables are not connected, but they are connected. This happens in single player as well as on the dedicated server.

![Farming Simulator 22 Screenshot 2023 03 13 - 19 51 46 16](https://user-images.githubusercontent.com/86984457/224803761-37b3a76a-d8fe-4bc1-a3d7-79c9dfe5211f.png)
![Farming Simulator 22 Screenshot 2023 03 13 - 19 52 10 89](https://user-images.githubusercontent.com/86984457/224803793-ed04f001-7fb1-46e0-88f9-06a964b244fd.png)

**Comment by stijnwop** [comment]
Are you sure you updated the mod? As this very much indicates the previous reported issue #159 

Else please list the mods/vehicles used and a reproducable path.

**Comment by Horschde** [comment]
Hello, I have now tested Elmcreek on a clean safegame with vanilla map.
I have tested some trailers with the standard Fendt "Black Beauty" and noticed that this problem has to do with the Strautmann trailers. I took several Strautmann from the Modhub and they all have the same phenomenon.

![Farming Simulator 22 Screenshot 2023 03 14 - 19 32 23 70](https://user-images.githubusercontent.com/86984457/225107588-253a60b4-d703-4d8c-8df4-859d7bacb783.png)
![Farming Simulator 22 Screenshot 2023 03 14 - 19 33 24 20](https://user-images.githubusercontent.com/86984457/225107654-e07ab899-d58b-4480-9ea4-5adc078cd604.png)


However, it only occurs with these trailers with V 2.2.0.0, when I test the same trailers with V 2.0.0.0, they drive at full speed.

![Farming Simulator 22 Screenshot 2023 03 14 - 19 38 01 98](https://user-images.githubusercontent.com/86984457/225108823-52b2a12e-f835-485a-95d5-0a189c94c72f.png)
![Farming Simulator 22 Screenshot 2023 03 14 - 19 41 30 39](https://user-images.githubusercontent.com/86984457/225108855-a021df7e-9928-4563-82eb-49623b497045.png)

I hope there is something useful in my information for you to narrow down the problem.

Sincerely yours
Horschd
